cmd/utils/fileIOUtils.go

```diff
@@ -42,7 +42,9 @@ func GetRemoteConfigFilePath() string {
 	if err != nil {
 		HandleErrorAndExit("Error getting user home directory: ", err)
 	}
-	remoteConfigFilePath := filepath.Join(userHomeDir, RemoteConfigFileName)
+	configDirectory := filepath.Join(userHomeDir, ConfigDirName)
```

Comment:
where did we define ".wso2micli" ?

Reply:
It was already defined inside Constants. But no usage. https://github.com/wso2/product-mi-tooling/blob/master/cmd/utils/constants.go#L30